libs/barista-components/experimental/combobox/src/combobox.ts

```diff
@@ -345,8 +348,10 @@ export class DtCombobox<T> extends _DtComboboxMixinBase
     );
     this._options.changes.pipe(startWith(null)).subscribe(() => {
```

Comment:
@bardosmisi I know it is not your code but please unsubscribe from this subscription :)

Reply:
Good catch, thanks.